Code/GraphMol/MolDraw2D/DrawMol.cpp

```diff
@@ -2889,11 +2895,32 @@ void DrawMol::bondNonRing(const Bond &bond, double offset, Point2D &l2s,
   const Atom *thirdAtom = nullptr;
   const Atom *fourthAtom = nullptr;
 
+  bool begTrunc = !atomLabels_[begAt->getIdx()];
+  bool endTrunc = !atomLabels_[endAt->getIdx()];
+
+  // find a neighbour of at1 that isn't at2 and if possible isn't directly
+  // opposite at1 to at2.
+  auto nonColinearNbor = [&](Atom *at1, Atom *at2) -> const Atom * {
```

Comment:
good work saving electrons with `Nbor`. ;-)
If this were in the public API I'd complain, but it's fine here